src/components/toolbar-menu/toolbarComponent.ts

```diff
@@ -73,6 +75,36 @@ export class ToolbarController {
   }
 
   /**
+   * Helper method for getting string value of {@link ToolbarType.BUTTON_SELECT}
+   * @memberof ToolbarController
+   * @function getToolbarListType
+   * @returns {string}
+   */
+  public getToolbarListType(): string {
+    return ToolbarType.BUTTON_SELECT.toString();
```

Comment:
This is just to access toolbarType enum from teplate, but I get it this is more [java based enum](https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html) we can always use [typescript's way](https://www.typescriptlang.org/docs/handbook/enums.html), cleaner and less code.